test/cql-pytest/test_materialized_view.py

```diff
@@ -1354,3 +1354,56 @@ def test_create_materialized_view_slash_name(cql, test_keyspace, table1):
     finally:
         # We shouldn't reach here, but if we did, let the test fail cleanly
         cql.execute(f'DROP MATERIALIZED VIEW IF EXISTS {test_keyspace}."/xyz/"')
+
+@pytest.fixture(scope="module")
+def mv1(cql, test_keyspace):
+    with new_test_table(cql, test_keyspace, 'p int, c int, x int, primary key (p)') as table:
+        with new_materialized_view(cql, table, '*', 'c, p', 'p is not null and c is not null') as mv:
+            yield mv
+
+# A materialized view may not be written to directly - it can only be written
+# through its base table. This test verifies that all CQL operations that can
+# write a table - INSERT, UPDATE, BATCH, DELETE, and TRUNCATE - are rejected
```

Comment:
Again, not an expert, IDK if `write a table` == `write to a table`.
```suggestion
# write to a table - INSERT, UPDATE, BATCH, DELETE, and TRUNCATE - are rejected
```

Reply:
I agree that it's better with "to".